### PR TITLE
Coin-M - Client, ExchangeInfoService and OrderService

### PR DESCRIPTION
**Pull request by alphayax**
Coin-M-Futures
- Add Client
- Add ExchangeInfoService
- Add OrderService

Related to #150 

This work is a basis inspired of futures files. Examples have been updated regarding biance documentation https://binance-docs.github.io/apidocs/delivery/en/#exchange-information

### CODE REVIEW

**Comment by afenrir** [comment]
Thanks for taking the time to add a large new feature, @alphayax. 

I haven't done a deep dive on it, but the first thing that caught my eye is that the package name is `coinM`. I understand why you've made that decision. However, camel case package names are not idiomatic Go. In my opinion, simply using `coinm` (or something else entirely) would be preferrable.

**Comment by alphayax** [comment]
> Thanks for taking the time to add a large new feature, @alphayax.
> 
> I haven't done a deep dive on it, but the first thing that caught my eye is that the package name is `coinM`. I understand why you've made that decision. However, camel case package names are not idiomatic Go. In my opinion, simply using `coinm` (or something else entirely) would be preferrable.

@afenrir  Thanks for your first return. I've renamed that into `coinm`, but maybe using `delivery` like in the binance docs make more sense... Feel free to give me your opinon.

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Name | Description | Status
 [wapi-api.md](https://github.com/binance-exchange/binance-official-api-docs/blob/master/wapi-api.md) | Details on the Withdrawal API (/wapi) | <input type="checkbox" checked>  Partially Implemented
 [margin-api.md](https://github.com/binance-exchange/binance-official-api-docs/blob/master/margin-api.md) | Details on the Margin API (/sapi) | <input type="checkbox" checked>  Implemented
 [futures-api.md](https://binance-docs.github.io/apidocs/futures/en/#general-info) | Details on the Futures API (/fapi) | <input type="checkbox" checked>  Partially Implemented
+[futures-api.md](https://binance-docs.github.io/apidocs/delivery/en/#general-info) | Details on the Coin-M Futures API (/dapi) | <input type="checkbox" checked>  Partially Implemented
```
Comment (adshao) [inline README.md:26]: it's better to use `delivery-api.md` instead of `futures-api.md`

File: README.md [hunk 2]
```diff
@@ -55,6 +56,7 @@ var (
 )
 client := binance.NewClient(apiKey, secretKey)
 futuresClient := binance.NewFuturesClient(apiKey, secretKey)
+coinMClient := binance.NewCoinMClient(apiKey, secretKey)
```
Comment (adshao) [inline README.md:59]: use deliveryClient instead of coinMClient

File: client.go [hunk 3]
```diff
@@ -177,6 +178,11 @@ func NewFuturesClient(apiKey, secretKey string) *futures.Client {
 	return futures.NewClient(apiKey, secretKey)
 }
 
+// NewCoinMClient initialize client for coin-M futures API
+func NewCoinMClient(apiKey, secretKey string) *delivery.Client {
```
Comment (adshao) [inline client.go:182]: s/NewCoinMClient/NewDeliveryClient

File: v2/client.go [hunk 4]
```diff
@@ -171,6 +172,11 @@ func NewFuturesClient(apiKey, secretKey string) *futures.Client {
 	return futures.NewClient(apiKey, secretKey)
 }
 
+// NewCoinMClient initialize client for coin-M futures API
+func NewCoinMClient(apiKey, secretKey string) *delivery.Client {
```
Comment (adshao) [inline v2/client.go:176]: s/NewCoinMClient/NewDeliveryClient

**Comment by adshao** [comment]
thanks @alphayax for this big feature

**Comment by alphayax** [comment]
@adshao Thanks for your review. All changes have been made and my comits have been rebased.

**Review by adshao — APPROVED** [review]

**Pull request merged**